README.md

```diff
@@ -1,5 +1,8 @@
 Authors Haven - A Social platform for the creative at heart.
 =======
+[![Build Status](https://travis-ci.org/andela/ah-backend-spaces.svg?branch=develop)](https://travis-ci.org/andela/ah-backend-spaces)
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-backend-spaces/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-backend-spaces?branch=develop)
+[![Maintainability](https://api.codeclimate.com/v1/badges/96588e5cfb3ba7da70bb/maintainability)](https://codeclimate.com/github/andela/ah-backend-spaces/maintainability)
 
```

Comment:
Same here

Reply:
No, it was not, I rebased with develop when I got merge conflicts after you had merged the two pending PR's to develop and so I got the .coveralls.yml file.

Reply:
@geofrocker, as regards this subject. @lytes20 is currently working on a new PR to solve this. I will then rebase once it is merged and be good to go. We are on it now.

Reply:
Great